bootstrap/web/iframe-adapter.js

```diff
@@ -183,27 +183,24 @@ var ZoweZLUX = {
         pluginDef: undefined,
         launchMetadata: undefined,
 
-        //True - Standalone, False - We are in regular desktop mode
+        //True - Single app mode, False - We are in regular desktop mode
         isSingleAppMode() {
             return new Promise(function(resolve, reject)  {
-                if (window.GIZA_SIMPLE_CONTAINER_REQUESTED) { //Ancient edgecase
+                if (window.top.GIZA_PLUGIN_TO_BE_LOADED) { //Ancient edgecase
```

Comment:
Why is window.top used in some places and window used in others?

Reply:
window.top is used here because zlux-app-manager creates these globals in the window parent. window.top is not used everywhere else in iframe-adapter because as my understanding, the events get emmitted as ZLUX events so people should listen to those anyway, not pollute the parent window further with stuff